src/UTF-unknown.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>net40;netstandard1.0;netstandard1.3;netstandard2.0;netcoreapp3.0</TargetFrameworks>
+    <TargetFrameworks>net462;net472;netstandard2.0;net6.0</TargetFrameworks>

```

Comment:
Why do you want to support net462/net472/net60 separately? You can leave your comment in this issue https://github.com/CharsetDetector/UTF-unknown/issues/145